src/architecture/messaging/cMsgCInterface/msg_C.h.in

```diff
@@ -0,0 +1,55 @@
+/* @MSG_AUTOSOURCE_LICENSE@ */
```

Comment:
File isn't actually new, but git thinks it is for some reason.

Reply:
Odd, I see what you mean.  But, this should be good to go though.